### ISSUE TITLE
feature request: User Roles

### ISSUE THREAD

**Issue by BrtCgn** [revoltchat/desktop#187]
### What do you want to see?

There should be an indicator to show the user's roles in the current server.

**Comment by sapioit** [comment]
I agree. And I think it should be on the user's profile, just like in Discord.
